vck/src/commonMain/kotlin/at/asitplus/wallet/lib/agent/Validator.kt

```diff
@@ -213,29 +223,41 @@ class Validator(
         if (!publicKey.matchesIdentifier(keyBinding.audience))
             return Verifier.VerifyPresentationResult.InvalidStructure(input)
                 .also { Napier.w("verifyVpSdJwt: Audience not correct: ${keyBinding.audience}") }
-        if (sdJwtResult.sdJwt.confirmationKey != null) {
+        @Suppress("DEPRECATION")
+        if (sdJwtResult.verifiableCredentialSdJwt.confirmationClaim != null) {
+            // TODO More general way to verify confirmation claim needed, as it may be a kid, jku, ...
```

Comment:
What is the hold-up here? If there is one, or it is just not prioritised, then this PR should be accompanied by an issue

Reply:
Disclussed offline